### PR TITLE
Update repositories.txt

### PR DESCRIPTION
**Pull request by adrian2793**
**Reason for removing this Library**

This library is a duplicate of [`Engine Control`](https://github.com/Arduino-Library-Collection/Engine-Control). I changed the name but I couldn't remove it from the list.

### CODE REVIEW

**Review by per1234 — COMMENTED** [review]
Hello @adrian2793. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

**Review by MatteoPologruto — APPROVED** [review]
Done:
```
2023/11/29 15:52:20 Loaded 6414 libraries from DB
Removing DC Motor Engine Control
Success!
```

**Pull request merged**
